editor/editor_plugin.h

```diff
@@ -137,7 +137,9 @@ class EditorPlugin : public Node {
 		CONTAINER_CANVAS_EDITOR_SIDE_LEFT,
 		CONTAINER_CANVAS_EDITOR_SIDE_RIGHT,
 		CONTAINER_CANVAS_EDITOR_BOTTOM,
-		CONTAINER_PROPERTY_EDITOR_BOTTOM
+		CONTAINER_PROPERTY_EDITOR_BOTTOM,
+		CONTAINER_PROJECT_SETTING_TAB_LEFT,
+		CONTAINER_PROJECT_SETTING_TAB_RIGHT
```

Comment:
If you amend the commit to take my above nitpick into account, could you also add a trailing comma so that future additions don't mark this line as changed needlessly?